src/libraries/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -249,6 +249,10 @@ namespace System.Drawing
 
         public static Color Purple => new Color(KnownColor.Purple);
 
+        /// <summary>
+        /// Gets a system-defined color that has an ARGB value of <c>#663399</c>.
+        /// </summary>
+        /// <value>A <see cref="Color" /> representing a system-defined color.</value>
```

Comment:
```suggestion
        /// <value>A system-defined color.</value>
```

No need to specify the type.